packages/react-shared-contexts/tsconfig.json

```diff
@@ -1,18 +1,25 @@
 {
   "extends": "../../tsconfig.base.json",
-  "include": ["src"],
   "compilerOptions": {
     "target": "ES2019",
-    "module": "CommonJS",
-    "lib": ["ES2019", "DOM"],
-    "outDir": "dist",
-    "jsx": "react",
-    "declaration": true,
-    "experimentalDecorators": true,
+    "noEmit": true,
+    "isolatedModules": true,
     "importHelpers": true,
+    "jsx": "react",
     "noUnusedLocals": true,
-    "preserveConstEnums": true,
-    "types": ["jest", "custom-global"],
-    "isolatedModules": true
-  }
+    "preserveConstEnums": true
+  },
+  "include": [],
+  "files": [],
+  "references": [
+    {
+      "path": "./tsconfig.lib.json"
+    },
+    {
+      "path": "./tsconfig.spec.json"
+    },
+    {
+      "path": "./.storybook/tsconfig.json"
+    }
```

Comment:
```suggestion
```

@Hotell does the generator script detect if a storybook tsconfig should be added or not ?

Reply:
Oh I see it adds `.storybook` to every package it tries to migrate. I guess @layershifter  deleted the storybook folder but not the reference to the tsconfig

Reply:
> I guess @layershifter deleted the storybook folder but not the reference to the tsconfig

Yup, missed this one. Thanks!